/**************************************************************************
File:         cnew_file_dlg.h
Author:       Song Xiaofeng
Date:         2016-05-04
Description:
**************************************************************************/
#ifndef CNEW_FILE_DLG_H
#define CNEW_FILE_DLG_H

#include <QDialog>
#include "ui_cnew_file_dlg.h"
//////////////////////////////////////////////////////////////////
///    Class:  CNewFileDlg
///    Brief:  
//////////////////////////////////////////////////////////////////
class CNewFileDlg : public QDialog
{
	Q_OBJECT
///////////////////////////////////////////////////////////////////
//all public functions are here
///////////////////////////////////////////////////////////////////
public:
	CNewFileDlg(QWidget *parent = 0);
	~CNewFileDlg();
///////////////////////////////////////////////////////////////////
//all signals are here
///////////////////////////////////////////////////////////////////
signals:
	void signal_new_file(const QString &name,const QString &type);
///////////////////////////////////////////////////////////////////
//all private functions are here
///////////////////////////////////////////////////////////////////
private:
	void mSetStyle();
///////////////////////////////////////////////////////////////////
//all private slots are here
///////////////////////////////////////////////////////////////////
private slots:
	void on_label_scenePix_linkHovered(const QString &link);
	void on_listWidget_clicked(const QModelIndex &index);
	void on_pushButton_ok_clicked();
///////////////////////////////////////////////////////////////////
//all properties  are here
///////////////////////////////////////////////////////////////////
private:
	Ui::CNewFileDlg ui;
};

#endif // CNEW_FILE_DLG_H
